genericapi/handler.go

```diff
@@ -100,6 +111,7 @@ func (h *Handler) ServeCreateAlert(w http.ResponseWriter, r *http.Request) {
 
 		var b struct {
 			Summary, Details, Action, Dedup *string
+			Meta                            alert.AlertMetaInput
```

Comment:
We don't have the same constraints here we do with GraphQL, so the simplest API would be just to require an object of key/value strings directly.

Since this is an API handler, using `map[string]string` instead of `alert.Metadata` here just serves to remove indirection so someone looking at this code can immediately see the expected type without having to switch context and look at the alert package.

```suggestion
			Meta                            map[string]string
```

Reply:
Okay👍
Q: Do you think the notification webhook should also send the meta values of type map[string]string?

Reply:
Yes, I think that makes the most sense, and makes it the easiest for downstream systems to use the data.

Reply:
okay